packages/connect-query-core/src/structural-sharing.test.ts

```diff
@@ -28,34 +28,41 @@ describe("structural sharing", () => {
     const oldData = create(schema, { sentence: "hi" });
     const newData = create(schema, { sentence: "hi" });
     const result = fn(oldData, newData);
-    expect(result).toStrictEqual(oldData);
+    expect(result).toBe(oldData);
```

Comment:
Switching all these to `toBe` so it can check referential integrity instead of deep equality.